### ISSUE TITLE
where has the mod gone?

### ISSUE THREAD

**Issue by DefomCode** [Exzotic5485/Create-Broken-Bad-Fabric#3]
The mod disappeared on modrinth. Why did this happen?

**Comment by Exzotic5485** [comment]
Got taken down for violating the rule of "Content must not promote any illegal activity, or advocate, promote or assist any unlawful act, including real-life drugs or illicit substances.". Though for some reason the forge / original mod is still up.

**Comment by DefomCode** [comment]
This is very strange and stupid... Maybe you will try to disguise the names or descriptions somehow and try again? It's strange that your post was deleted and the original was not, when the port had less than 1000 downloads, if I remember correctly

**Comment by Exzotic5485** [comment]
Yeah I'm not sure. You can still download it on curseforge or github releases for now though :)

**Comment by encode42** [comment]
I've created a thread with the content moderators. I've been against Modrinth's stance regarding mods deemed "13+", and am now pushing for change. It shouldn't be difficult for them to implement a warning or age restriction similar to Reddit's "NSFW" posts. However, I'm unsure if anything will actually come of this, since I've debated the rules directly with the developers in the past to no avail.

**Comment by encode42** [comment]
welp, that was fast

![image](https://github.com/Exzotic5485/Create-Broken-Bad-Fabric/assets/34699884/07927ff7-220d-4d69-9f9c-c9a998de238f)

It really seems counter-intuitive for them to compete with CurseForge, yet shun certain mods from using their platform. During my time as a Modrinth content moderator, I've been told to allow mods that I personally deem worse than this in morals. It seems they've fallen into a habit that many open-source projects suffer from - the urge to close issues, fast. I'll probably move my suggestion to one of their GitHub repositories, where hopefully the same doesn't happen.
